src/d2_docker/docker-compose.yml

```diff
@@ -32,6 +34,8 @@ services:
             POSTGRES_DB: dhis2
             POSTGRES_USER: dhis
             POSTGRES_PASSWORD: dhis
+        ports:
+            - "${DB_DEBUG_PORT:-5432}:5432"
```

Comment:
Right now we are exposing port 5432 always, which we shouldn't but it's not easy to conditionally forward a port